a3p-integration/README.md

```diff
@@ -99,11 +113,109 @@ make -C ../packages/deployment docker-build-sdk
 
 Some core-eval proposals `submission` content are generated from the `agoric-sdk`
 code, and must be rebuilt every time there is a change. The
-`scripts/generate-a3p-submission.sh` script contains commands to generate the
+`scripts/generate-a3p-submissions.sh` script contains commands to generate the
 core-eval content and move it to the expected proposal package's submission
 directory. It is executed as part of `a3p-integration`'s `build:submissions` step.
-Each proposal that requires such a build step should add an `sdk-generate` property
-in its `agoricProposal` config.
+Each proposal that requires such a build step should add an entry to the
+`sdk-generate` array in the `agoricProposal` section of `package.json`.
+
+Submissions that don't need to pass in options or generate references to source
+bundles can be written directly in a `foo-submission` subdirectory of the
+proposal. These would incude a .js or .tjs file, accompanied by a similarly-named
+-permit.json file that can contain just `true`, or a complete permission manifest.
+The submission should return the script, which can take BootstrapPowers as a
+parameter.  
+
+If the submission does require a bundle reference, or other options to be
+provided, it should be written as two parts. The proposal usually goes in
+`.../vats/proposals`, and the script in `.../builders/scripts/vats`.
+
+### Proposal
```

Comment:
From here down is not about a3p. I think it should go in `deploy-script-support`. cc @michaelfig @dckc for input.